nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/queue/state/queue-listing/index.ts

```diff
@@ -102,7 +102,7 @@ export interface FlowFileDialogRequest {
 
 export interface QueueListingState {
     activeListingRequest: ListingRequest | null;
-    completedListingRequest: ListingRequest | null;
+    completedListingRequest: ListingRequest;
```

Comment:
Let me spend some time re-evaluating. Rather than being null, I provided an empty state in the provenance and lineage reducers similar to how the canvas state is initialized in the flow reducer. I should be able to make this change but may need further adjustments where this state is consumed.